### PR TITLE
Create azure_app_credential_added.yml

### PR DESCRIPTION
**Pull request by MarkMorow**
App Credential Add rule

### CODE REVIEW

**Comment by MarkMorow** [comment]
@frack113 ok I think it's fixed.

File: rules/cloud/azure/azure_app_credential_added.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title: Added credentials to existing application
```
Comment (frack113) [inline rules/cloud/azure/azure_app_credential_added.yml:1]: `title: Added Credentials to Existing Application` https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#common-pitfalls

File: rules/cloud/azure/azure_app_credential_added.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  service: auditlogs
+detection:
+  selection:
+    properties.message: 
+      - Update Application-Certificates and secrets management
+      - Update Service principal/Update Application
+  condition: selection
+falsepositives:
+  - When credentials are added/removed as part of the normal working hours/workflows
+level: High
```
Comment (frack113) [inline rules/cloud/azure/azure_app_credential_added.yml:19]: `level: high` https://github.com/SigmaHQ/sigma/wiki/Specification#level

File: rules/cloud/azure/azure_app_credential_added.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+status: expiramental
```
Comment (frack113) [inline rules/cloud/azure/azure_app_credential_added.yml:20]: `status: experimental` https://github.com/SigmaHQ/sigma/wiki/Specification#status-optional

**Review by frack113 — CHANGES_REQUESTED** [review]

**Comment by MarkMorow** [comment]
@frack113 ok sorry, added a bit more after reviewing your guide. Thanks!

**Review by frack113 — APPROVED** [review]

**Pull request merged**